sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -586,15 +603,16 @@ static const char *getDeviceLibExtensionStr(DeviceLibExt Extension) {
 static RT::PiProgram loadDeviceLibFallback(
     const ContextImplPtr Context, DeviceLibExt Extension,
     const RT::PiDevice &Device,
-    std::map<std::pair<DeviceLibExt, RT::PiDevice>, RT::PiProgram>
-        &CachedLibPrograms) {
+    std::map<std::pair<DeviceLibExt, RT::PiDevice>,
+             std::pair<RT::PiProgram, string_class>> &CachedLibPrograms) {
 
   const char *LibFileName = getDeviceLibFilename(Extension);
-  auto CacheResult = CachedLibPrograms.emplace(
-      std::make_pair(std::make_pair(Extension, Device), nullptr));
+  // TODO: Get build options
```

Comment:
Could you please elaborate this TODO a bit?

Reply:
No TODO is needed here.